commands.go

```diff
@@ -28,8 +28,9 @@ const (
 	deleteUserMessageType = messaging.MessageType(0x32)
 
 	// Shard messages
-	createShardGroupIfNotExistsMessageType = messaging.MessageType(0x40)
-	deleteShardGroupMessageType            = messaging.MessageType(0x41)
+	createShardGroupIfNotExistsMessageType    = messaging.MessageType(0x40)
```

Comment:
OK, I see.

Is there some way to collapse both of these operations into 1 operation? Why are they different? Why not move to a design such that when a retention policy is first created, create the initial shard groups then, and wait for that to complete. And from then one, background creation happens.